extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/impl/RecordPartImpl.java

```diff
@@ -22,10 +22,12 @@
 
 import javax.annotation.Nonnull;
 import java.io.IOException;
+import java.io.Serializable;
 import java.util.Map;
 import java.util.Objects;
 
-class RecordPartImpl implements RecordPart {
+class RecordPartImpl implements RecordPart, Serializable {
+    private static final long serialVersionUID = 1L;
```

Comment:
Why does it need to be Serializable?

Reply:
No longer needed, connected to CdcSinksTest - now it's done on suppliers there and Serializable is not necessary